exes/mqtt-broadcaster/inc/config.hpp

```diff
@@ -1,30 +1,56 @@
 #pragma once
 
 #include <tfc/confman.hpp>
-#include <tfc/confman/observable.hpp>
+
+namespace tfc::mqtt {
+
+struct signal_defintion {
+  std::string name{};
```

Comment:
maybe we should store description here as optional field and pass to the ipc signal constructor?